app/pages/index.tsx

```diff
@@ -4,7 +4,16 @@ import { WithRedux } from "components/WithRedux";
 import { NextPage } from "next";
 import { PageHead } from "components/PageHead";
 import { IS_PRODUCTION } from "lib/constants";
+import { messages as default_messages } from "../locale/en/messages";
+import { i18n } from "@lingui/core";
 
+export async function getStaticProps() {
+  i18n.load("en", default_messages);
+  i18n.activate("en");
```

Comment:
So this means non-english pages will see a flash of english text. Better than the string IDs I guess! We don't have any other option since we are treating this like a static Single Page App.